autopush/settings.py

```diff
@@ -168,7 +170,12 @@ def update(self, **kwargs):
         ``self.fernet`` attribute will be initialized"""
         for key, val in kwargs.items():
             if key == "crypto_key":
-                self.fernet = Fernet(val)
+                fkeys = []
+                if not isinstance(val, type([])):
```

Comment:
Less than fun fact: pdb chokes on isinstance(val, list) saying that local variable "list" is not defined, thus the "type([])" nonsense. I can add a comment if you like.